### ISSUE TITLE
Current Summary and Problems I will need to fix.

### ISSUE THREAD

**Issue by telbohtimy** [CMPUT301W14T04/Team04Project#10]
activity_create_comment.xml- I made and xml file for the create comments all that it currently has is a textbox for writing and a create button.
CreateComment.java- The intention of the class is to take the text box and take the current date and current username and create a comment class with all the needed information
 -issues with this class :
*EditText text = (EditText) findViewById(R.id.comment);
*setContentView(R.layout.activity_create_comment);
*getMenuInflater().inflate(R.menu.create_comment, menu);
Those three lines are giving me errors not sure why but my belief it is because the mainactivity has not sent an intent to them yet.
-Intentions for future- I want to be able to send the comment class through an intent and add it to an array of classes on the browse comment page.
Comments.java- just added the getters and setters for the information the comment needs
Dates.java- gets the current date and puts it in the format "Month day, year- hour:minute"
User.java-This class really just holds the username of the app user
activity_options.xml- an xml page that'll dynamically create a text saying "your username is:" or "you have not made a username" and a button that is also dynamically created with "change" and "create"
 -I dynamically created them just for the different messages but if you believe they are unnecessary then i'll come up with a general message.

options.java- dynamically creates the textview and button and also retreats the username from internal storage if its there
-intentions-I want to be able to store all the different usernames online to double check that no two people have the same username
-questions-would there be an option to delete their account?
-issues-*setContentView(R.layout.activity_options);
*getMenuInflater().inflate(R.menu.options, menu);
*LinearLayout ll = (LinearLayout)findViewById(R.id.optionId);
these lines also give me errors but I believe it is for the same reason as CreateComment.java class.
If you have any concerns, questions, comments, disagreements, or requests please notify me.

**Comment by gramirezualberta** [comment]
Hi 

```
EditText text = (EditText) findViewById(R.id.comment);
*setContentView(R.layout.activity_create_comment);
*getMenuInflater().inflate(R.menu.create_comment, menu); 
```

these errors could be because there is not a R.java generated in eclipse. This happen mainly because there is error in XML file, so i suggest before pushing any changes to the Team repository, create a branch or fork the project, so that we have a clean project to work on. I will see if I can solve those errors. 

Location file. I renamed Location to Locations, so there is not overlap between the Android Location class, and I don't see why do we have a Location class. 

I did change user.java.

Options.java I think is better to identify the user by unique ID than by userName. 

_questions-would there be an option to delete their account?_

The requirements do not say anything about creating an account, so i guess we don't actually need to delete anything .
